DepositeCalcTests/Pages/CalculatorPage.cs

```diff
@@ -112,5 +115,22 @@ public bool IsOpened()
         {
             return driver.Url.Contains("Calculator");
         }
+
+        public HistoryPage OpenHistory()
+        {
+            HistoryLnk.Click();
+            return new HistoryPage(driver);
+        }
+
+        public void Open()
+        {
+            driver.Url = "https://localhost:5001/Calculator";
+        }
+
+        public CalculatorPage WeitForReady()
+        {
+            new WebDriverWait(driver, TimeSpan.FromSeconds(5)).Until(_ => IsOpened());
+            return new CalculatorPage(driver);
+        }
```

Comment:
This page action is 'wait', it doesn't need to return new page instance.
```suggestion
        public void WeitForReady()
        {
            new WebDriverWait(driver, TimeSpan.FromSeconds(5)).Until(_ => IsOpened());
        }
```